code/citadel/organs/genitals_sprite_accessories.dm

```diff
@@ -27,15 +27,57 @@
 	icon_state = "tapered"
 	name = "Tapered"
 
+//Vaginas
+/datum/sprite_accessory/vagina
+	icon = 'code/citadel/icons/vagina_onmob.dmi'
+	icon_state = null
+	name = "vagina"
+	gender_specific = 0
+	color_src = "vag_color"
+	locked = 0
+
+/datum/sprite_accessory/vagina/human
+	icon_state = "human"
+	name = "Human"
+
+/datum/sprite_accessory/vagina/gaping
+	icon_state = "gaping"
+	name = "Gaping"
+
+//I think this one is intended for arousal... but it would only work for human vaginas in this case...
+//So I'm leaving it as a selectable option until we can make/need variants for the sprites to change based on arousal.
+/datum/sprite_accessory/vagina/dripping
+	icon_state = "dripping"
+	name = "Dripping"
+
+/datum/sprite_accessory/vagina/tentacles
+	icon_state = "tentacle"
+	name = "Tentacle"
+
+/datum/sprite_accessory/vagina/dentata
+	icon_state = "dentata"
+	name = "Dentata"
+
+/datum/sprite_accessory/vagina/hairy
+	icon_state = "hairy"
+	name = "Hairy"
```

Comment:
✅ Disputed, mostly because it doesn't fit with most of the sprites. If we want customisable hair-styles as some sort of additional overlay, we'll need some sprites for that too. As it is, I'm only adding this in as a shape, because that's what the icon we got is drawn as (it would completely cover most of the other sprites, making them mutually exclusive anyway).